specs/api.kuflow.com/v2024-06-14/openapi.yaml

```diff
@@ -234,6 +240,39 @@ paths:
         default:
           $ref: "#/components/responses/DefaultError"
 
+  /groups:
+    get:
+      summary: Find all accessible Groups
+      description: |
+        List all the Groups that have been created and the used credentials has access.
+
+        Available sort query values: id, name
+      operationId: findGroups
+      tags:
+        - apiRestExternalV20240614Groups
+        - group
+      parameters:
+        - $ref: "#/components/parameters/SizeQueryParam"
+        - $ref: "#/components/parameters/PageQueryParam"
+        - $ref: "#/components/parameters/SortQueryParam"
+        - name: type
+          description: Filter groups by type.
+          in: query
+          schema:
+            $ref: "#/components/schemas/GroupType"
+        - $ref: "#/components/parameters/GroupIdQueryParam"
```

Comment:
Is this ok?

Or maybe we must create a GET /groups/{ID} ??

Reply:
For now we don't need to create groups